app/views/hub/users/edit_role.html.erb

```diff
@@ -10,41 +10,59 @@
 
   <%= form_for(@user, method: :patch, url: update_role_hub_user_path, local: true, builder: VitaMinFormBuilder) do |f| %>
 
-    <%= f.hidden_field :role, value: params.dig(:user, :role) %>
-    <% if params.dig(:user, :role) == AdminRole::TYPE %>
+    <%= f.hidden_field :role, value: params[:role] %>
```

Comment:
The nested {user: { role: } param didn't really serve a functional purpose, so I simplified it down to just coming in as params[:role] without nesting in all implementations.